src/Symfony/Component/PropertyAccess/Tests/PropertyAccessorTest.php

```diff
@@ -52,6 +52,26 @@ protected function setUp(): void
         $this->propertyAccessor = new PropertyAccessor();
     }
 
+    public function testPrefersPropertyOverMethodWithSameNameAndRequiredArgs()
+    {
+        $obj = new class {
+            public readonly bool $loaded;
+
+            public function __construct()
+            {
+                $this->loaded = true;
+            }
```

Comment:
I suppose we don't need this boilerplate?
```suggestion
            public bool $loaded = true;
```